app/client/stylesheets/_navbar.scss

```diff
@@ -0,0 +1,187 @@
+/* Keyframes */
+@keyframes spin {
+	100% {
+		transform:rotate(360deg);
+	}
+}
+
+/* Navbar */
+#navbar {
+	width: 100%;
+	z-index: 999;
+	font-family: 'Rubik', sans-serif;
+	color: white;
+	font-weight: 300;
+	font-size: 0.8em;
+	letter-spacing: 0.07em;
+
+	.navbar {
+		width: 100%;
+		height: 100px;
+		padding: 35px 50px;
+		background: linear-gradient(-225deg, #2CD8D5 0%, #6B8DD6 48%, #8E37D7 100%);
+		display: flex;
+		animation: fadein .5s both;
+		animation-delay: .1s;
+	}
+
+	.navbar-home {
+		position: absolute;
+		background: none;
+		padding: 35px 50px 0 50px;
+		transition: all .2s ease;
+
+		.logo-header {
+			margin-top: 0;
+		}
+	}
+
+	.logo-header {
+		animation: fadein .5s both;
+		animation-delay: .15s;
+		height: calc(100% - 35px);
+		position: absolute;
+		margin-top: -17.5px;
+		transition: all .2s ease;
+	}
+
+	.logo-cog {
+		animation: spin 12s linear infinite;
+	}
+
+	.logo-header>img {
+		height: 100%;
+	}
+
+	.logo-header>span {
+		font-weight: 700;
+		font-size: 1.5em;
+		padding-left: 0.5em;
+		letter-spacing: 0.02em;
+		color: white;
+		margin-top: -1px;
+	}
+
+	.navbar-menu {
+		display: flex;
+		flex: 1 0 auto;
+		justify-content: flex-end;
+		align-items: center;
+		margin: 0;
+	}
+
+	.menu-item {
+		color: rgba(255,255,255,0.95);
+		cursor: pointer;
+		transition: all .2s ease;
+	}
+
+	.menu-item+.menu-item:not(.menu-item--sponsor) {
+		padding-left: 3%;
+	}
+
+	.menu-item:hover {
+		color: rgba(255,255,255,1);
+	}
+
+	.menu-item--sponsor {
+		border: 1px solid rgba(255,255,255,0.95);
+		border-radius: 25px;
+		padding: 10px 25px;
+		margin-left: 2.5%;
+		background-color: transparent;
+		transition: all .2s ease;
+	}
+
+	.menu-item--sponsor:hover {
+		background-color: white;
+		color: #6B8DD6;
+		border: 1px solid rgba(255,255,255,1);
+	}
+
+	.navbar-mobile {
+		width: 100%;
+		height: 70px;
+		padding: 10px 30px;
+		background: linear-gradient(-225deg, #2CD8D5 0%, #6B8DD6 48%, #8E37D7 100%);
+		display: flex;
+		animation: fadein .2s both;
+		transition: all .2s ease;
+
+		.logo-header {
+			margin-top: 0;
+			height: 50px;
+			left: 0;
+			right: 0;
+			text-align: center;
+		}
+
+		.navbar-mobile-menu-button {
+			display: flex;
+			flex: 1 0 auto;
+			justify-content: flex-end;
+			align-items: center;
+			padding: 0;
+			font-size: 2em;
+			z-index: 999;
+		}
+	}
+
+	.navbar-mobile-transparent {
+		background: none;
+	}
+
+	/* Mobile */
+	@media (max-width: 767px) {
+		.navbar {
+			justify-content: center;
+		}
+	}
+}
+
+.ui.modals {
+	padding: 0;
+}
+
+#navbar-modal {
```

Comment:
Same here, any specific reason to use Id instead of class?

Reply:
Yep, gotta use CSS specificity rules to override Semantic UI's properties